torch/_inductor/fx_passes/group_batch_fusion.py

```diff
@@ -310,7 +337,15 @@ def match(self, node: torch.fx.Node):
             alpha = node.kwargs.get("alpha", 1.0)
```

Comment:
Would be nice to generalize to get non-Tensor values by looking at the schema instead of hardcoding alpha, and rounding_mode.

Reply:
Can I have more context on schema? Do you mean by looking at its args and kwargs?

Reply:
Right, looking at schema args and kwargs. You can call `torch.fx.operator_schemas.normalize_function` with `normalize_to_only_use_kwargs=True` and then hash on all the non-tensor args to generalize the `rounding_mode = node.kwargs.get("rounding_mode", None)` code. Not needed for this pr..

Reply:
In this fusion pass, we combine all pointwise ops, for example, torch.add and torch.mul. The problem is not all pointwise op has the "alpha" and "rounding_mode" keywords. However, in order to generalize to all pointwise ops, we need to assign some "default" values to the ops that do not have them. It seems like normalize_function cannot resolve the hardcode requirements.